zerver/tests/test_slack_importer.py

```diff
@@ -374,7 +374,7 @@ def test_users_to_zerver_userprofile(self, mock_get_data_file: mock.Mock) -> Non
             "Xf06054BBB": {"value": "random2"},
             "Xf023DSCdd": {"value": "employer"},
         }
-        user_data = [
+        user_data: list[dict[str, Any]] = [
```

Comment:
type hint is added to this variable because Mypy was complaining when I tampered with the users' emails for the new tests just a few lines below.